docs/install.md

```diff
@@ -18,12 +18,9 @@ First, install Docker version 1.3 or greater:
 
 ### Install Compose
 
-To install Compose, run the following commands:
+To install Compose, run the following command:
 
-    curl -L https://github.com/docker/compose/releases/download/1.2.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
-    chmod +x /usr/local/bin/docker-compose
-
-> Note: If you get a "Permission denied" error, your `/usr/local/bin` directory probably isn't writable and you'll need to install Compose as the superuser. Run `sudo -i`, then the two commands above, then `exit`.
+    sudo bash -c "curl -L https://github.com/docker/compose/releases/download/1.2.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose && chmod +x /usr/local/bin/docker-compose"
```

Comment:
I'm using MacPorts as many others who do not prefer do have their package
manager rely on OS and have clean separation. Same for users on Linux.

The only better way than the one proposed, could be the original Brew/Ports
package so one could install it from there. Or have RVM/NVM-like
installation requiring path update.


-- 
Sent from Gmail Mobile